src/SilverStripe/BehatExtension/Context/BasicContext.php

```diff
@@ -942,4 +966,14 @@ public function spin($lambda, $wait = 60) {
             $backtrace[1]['function']
         ));
     }
+	
+	
+	
+	/**
+	 * We have to catch exceptions and log somehow else otherwise behat falls over
+	 */
+	protected function logException($e){
+		file_put_contents('php://stderr', 'Exception caught: '.$e);
```

Comment:
Right, confirmed that toString() does something useful: http://php.net/manual/en/exception.tostring.php - good stuff.